File: programcki/decrypt/Decrypt.py
def main():
    f = open("cypher.txt")
    vsebina = f.read();
    vsebina = vsebina.strip().split(" ")
    print(vsebina)
    beseda2pojavitev = dict()
    for beseda in vsebina:
        if beseda in beseda2pojavitev:
            beseda2pojavitev[beseda] += 1
        else:
            beseda2pojavitev[beseda] = 1
    f.close()
    print(beseda2pojavitev)

main()

File: programcki/decrypt/Decrypt.java
import java.io.File;
import java.util.Scanner;
import java.util.HashMap;
import java.util.Arrays;
import java.io.FileNotFoundException;

public class Decrypt {
    public static void main(String[] args) {
        try {
            Scanner cypher = new Scanner(new File("cypher.txt"));
            HashMap<String, Integer> word2occurances = new HashMap<String, Integer>();
            while (cypher.hasNext()) {
                String beseda = cypher.next();
                int count = word2occurances.containsKey(beseda) ? word2occurances.get(beseda) : 0;
                word2occurances.put(beseda, count + 1);
            }
            System.out.println(Arrays.asList(word2occurances));
        }
        catch (FileNotFoundException e) {
            System.out.println("File not found");
        }
    }
}